packages/core/src/queue/event-queue.ts

```diff
@@ -232,16 +223,24 @@ export class EventQueue extends Emitter<EventQueueEmitterContract> {
     return true
   }
 
-  private availableExtensions(denyList: Integrations): CorePluginsByType {
+  private availableExtensions(denyList: Integrations) {
     const available = this.plugins.filter((p) => {
       // Only filter out destination plugins or the Segment.io plugin
       if (p.type !== 'destination' && p.name !== 'Segment.io') {
         return true
       }
 
```

Comment:
This code was recently added to the browser event queue by neek for the destination actions work, just moving it here.